beacon_node/beacon_chain/src/builder.rs

```diff
@@ -832,10 +832,14 @@ where
         let genesis_backfill_slot = if self.chain_config.genesis_backfill {
             Slot::new(0)
         } else {
+            #[cfg(not(test_backfill))]
```

Comment:
should be `#[cfg(not(feature = "test_backfill")]` right?
btw, nit: you can do:
```suggestion
            let backfill_epoch_range = if cfg!(feature = "test_backfill") {
                3
            } else {
                (self.spec.min_validator_withdrawability_delay + self.spec.churn_limit_quotient)
                    .as_u64()
                    / 2
            };

```